code/modules/clothing/clothing.dm

```diff
@@ -76,21 +76,21 @@
 		var/atom/movable/screen/inventory/hand/H = over_object
 		if(M.putItemFromInventoryInHandIfPossible(src, H.held_index))
 			add_fingerprint(usr)
-
-/obj/item/reagent_containers/food/snacks/clothing
+///This code is cursed, moths are cursed, and someday I will destroy it. but today is not that day.
```

Comment:
As punishment for making fun of my code here is a nitpick:

```suggestion
//This code is cursed, moths are cursed, and someday I will destroy it. but today is not that day.
```

This isn't a comment fit for the autodocs since it doesn't describe what `/obj/item/food/clothing` is (other than cursed, of course), so it should not be written as a documentation comment.